src/BinaryParsers/PEBinary/PEBinary.cs

```diff
@@ -80,7 +80,20 @@ public static void ClearLocalSymbolDirectoriesCache()
 
         public PE PE { get; private set; }
 
-        public Pdb Pdb => this.pdb?.Value;
+        public Pdb Pdb 
+        {
+            get
+            {
+                try
+                {
+                    return this.pdb?.Value;

```

Comment:
>pdb [](http://example.com/codeflow?start=32&length=3)

this doesn't look right to me. first, the code is hard to read, why would a field retrieval raise an exception?

the answer is that the field is itself lazily initialized. ok. :)

so my question is why you don't update LoadPdb instead to return null for this case.